microreactors/hpmr/steady/neutronics_eigenvalue.i

```diff
@@ -0,0 +1,277 @@
+# ##################################################################################################################
+# AUTHORS:            Stefano Terlizzi and Vincent Labouré
+# DATE:               June 2023
+# ORGANIZATION:       Idaho National Laboratory (INL), C-110
+# CITATION:           S. Terlizzi and V. Labouré. (2023).
+#                     "Asymptotic hydrogen redistribution analysis in yttrium-hydride-moderated heat-pipe-cooled
+#                     microreactors using DireWolf", Annals of Nuclear Energy, Vol. 186, 109735.
+# DESCRIPTION:        Main input file containing neutronic model for the SiMBA reactor.
+# FUNDS:              This work was supported by INL Laboratory Directed Research&
+#                     Development (LDRD) Program under U.S. Department of Energy (DOE) Idaho Operations Office,
+#                     United States of America contract DE-AC07-05ID14517.
+# ##################################################################################################################
+# Block assignment
+bison_fuel_blocks = '1 2'
+moderator_blocks = '3 4'
+monolith_blocks = '8 22' # For now fill the air gap with monolith
+reflector_blocks = '10 11 13 14 15 16 17 18 19'
+absorber_blocks = '12'
+bison_mod_blocks = '${moderator_blocks} ${monolith_blocks}'
+bison_ref_blocks = '${reflector_blocks} ${absorber_blocks}'
+[Mesh]
+  [main]
+    type = FileMeshGenerator
+    file = '../mesh/one_twelfth_empire_core_in.e'
+  []
+  [coarse_mesh]
+    type = FileMeshGenerator
+    file = '../mesh/one_twelfth_empire_core_CM_in.e'
+  []
+  [mesh]
+    type = CoarseMeshExtraElementIDGenerator
+    input = main
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+[PowerDensity]
+  power = 1.66666e5 # power: 2e6 / 12 = 1.66666e5 W/m
+  power_density_variable = power_density
+  integrated_power_postprocessor = integrated_power
+[]
+
+[TransportSystems]
+  equation_type = eigenvalue
+  particle = neutron
+  G = 11
+
+  ReflectingBoundary = 'bottom topleft'
+  VacuumBoundary = 'right back front'
+
+  [sn]
+    scheme = DFEM-SN
+    n_delay_groups = 6
+    family = MONOMIAL
+    order = FIRST
+    AQtype = Gauss-Chebyshev
+    NPolar = 1 # use >=2 for final runs (4 sawtooth nodes sufficient)
+    NAzmthl = 3 # use >=6 for final runs (4 sawtooth nodes sufficient)
```

Comment:
this would be nice to include in the doco (that we use more to get our results)

Reply:
works for me

Reply:
comments removed